integration-tests/browseSolutionsPage.test.js

```diff
@@ -0,0 +1,27 @@
+import nock from 'nock';
+import { Selector } from 'testcafe';
+import aSolutionList from './fixtures/aSolutionList.json';
+
+const mocks = () => {
+  nock('http://localhost:8080')
+    .get('/api/v1/Solutions')
+    .reply(200, aSolutionList);
+};
+
+const pageSetup = async (t) => {
+  mocks();
+  await t.navigateTo('http://localhost:1234/browse-solutions');
+};
+
+fixture('Browse Solutions Page');
+
+test('should navigate to browse solutions page when click Go back', async (t) => {
+  pageSetup(t);

```

Comment:
Shouldn't this go back to the home page? I think the description of the test is wrong here